README.md

```diff
@@ -1,163 +1,183 @@
 # Miden Virtual Machine
 
-<a href="https://github.com/0xPolygonMiden/miden-vm/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
-<img src="https://github.com/0xPolygonMiden/miden-vm/workflows/CI/badge.svg?branch=main">
-<img src="https://img.shields.io/badge/rustc-1.75+-lightgray.svg">
-<a href="https://crates.io/crates/miden-vm"><img src="https://img.shields.io/crates/v/miden-vm"></a>
+[![LICENSE](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/0xPolygonMiden/miden-vm/blob/main/LICENSE)
+[![test](https://github.com/0xPolygonMiden/miden-vm/actions/workflows/test.yml/badge.svg)](https://github.com/0xPolygonMiden/miden-vm/actions/workflows/test.yml)
+[![build](https://github.com/0xPolygonMiden/miden-vm/actions/workflows/build.yml/badge.svg)](https://github.com/0xPolygonMiden/miden-vm/actions/workflows/build.yml)
+[![RUST_VERSION](https://img.shields.io/badge/rustc-1.78+-lightgray.svg)](https://www.rust-lang.org/tools/install)
+[![GitHub Release](https://img.shields.io/github/release/0xPolygonMiden/miden-vm)](https://github.com/0xPolygonMiden/miden-vm/releases/)
```

Comment:
I think this is redundant with the next line. So, I'd remove it.

Reply:
I left both because the github release here is not the same as the crates io release.

Reply:
I think that's OK - having two versions may be more confusing that helpful.